components/cam/bld/configure

```diff
@@ -266,6 +266,21 @@ OPTIONS
                         perl \$OSNAME variable.
      -usr_src <dir1>[,<dir2>[,<dir3>[...]]]
                         Directories containing user source code.
+
+  Options for MMF CAM:
```

Comment:
I'd lose the "CAM" here. Replace with "EAM".

Reply:
will do